src/main/java/org/jabref/logic/protectedterms/ProtectedTermsPreferences.java

```diff
@@ -2,34 +2,73 @@
 
 import java.util.List;
 
+import javafx.beans.property.ListProperty;
+import javafx.beans.property.SimpleListProperty;
+import javafx.collections.FXCollections;
+import javafx.collections.ObservableList;
+
 public class ProtectedTermsPreferences {
 
-    private final List<String> enabledInternalTermLists;
-    private final List<String> enabledExternalTermLists;
-    private final List<String> disabledInternalTermLists;
-    private final List<String> disabledExternalTermLists;
+    private final ListProperty<String> enabledInternalTermLists;
```

Comment:
Observable lists should be enough. list properties are only needed if you want to change the list in itself.

Reply:
This is from #8046. @calixtus please keep this in mind.